README.md

```diff
@@ -1,2 +1,31 @@
 # hello-relativepath
-introduction to express.js
+
+
+## Introduction
+
+The purpose of the first iteration of this project is to run a basic express.js API. 
+
+In order to run this first iteration you must have nodejs and express installed. 
+
+## Install Dependencies
```

Comment:
:+1:

Reply:
README shrunk down, now Deploy is copy a command and paste it into your shell.

Once I upload the hello-world-deploy.sh (after we get through the README) I can point that command to an actual path.

side note: I wrote and tested all of the shell script in a docker container already (and uploaded it to my server so I could confirm the readme command works).